build.gradle

```diff
@@ -191,8 +196,61 @@ task copyXwsetversion(type: Copy) {
     into(distBin + '/db-maintenance')
 }
 
+task copyXwhepgenkeys(type: Copy) {
+
+    into(dist+"/bin")
+
+    into(".") {
+        from(scripts)
+        include('xwhepgenkeys.template')
+        rename('xwhepgenkeys.template', 'xwhepgenkeys')
+        include('xwhepupdateopenidcerts.template')
+        rename('xwhepupdateopenidcerts.template', 'xwhepupdateopenidcerts')
+
+        def props = new Properties()
+        file(resources+"/conf"+'/xwconfigure.values').withInputStream {
+            props.load(it)
+        }
+
+        Set<String> keys = props.stringPropertyNames();
+        for (String key : keys) {
+            props.setProperty(key, props.getProperty(key).replace('\'',''));
+        }
+
+        def certname = "cn=" + props.CERTCN + ",o=" + props.CERTO + ",c=" + props.CERTC
+        if (props.CERTOU != "") {
+            certname += ",ou=" + props.CERTOU
+        }
+        if (props.CERTL != "") {
+            certname += ",l=" + props.CERTL
+        }
+
+        filter(ReplaceTokens, tokens: [CERTNAME: certname], beginToken: '$', endToken: '')
```

Comment:
Maybe we should make the token standard. Let's use \@ instead of dollar or other token.